content/microservices/grpc.md

```diff
@@ -158,7 +160,7 @@ Both `@GrpcMethod()` decorator arguments are optional. If called without the sec
 @Controller()
 export class HeroesController {
   @GrpcMethod('HeroesService')
-  findOne(data: HeroById, metadata: any): Hero {
+  findOne(data: HeroById, metadata: any, call: any): Hero {
```

Comment:
Can we use types for `metadata` and `call`? I believe that gRPC package exposes some type definitions we could import here

Reply:
We can use Metadata from grpc.Metadata, but idk what type for call.

Reply:
It depends on the type of handler. 

For a unary call (by default), it should be `grpc.ServerUnaryCall<RequestType>`.
For client streaming - `grpc.ServerReadableStream<RequestType>`
For server streaming - `grpc.ServerWritableStream<RequestType>`
For bidi streaming (duplex) - `grpc.ServerDuplexStream<RequestType, ResponseType>`

We could just use `any` for `RequestType`/`ResponseType` type arguments to slightly simplify this.

Reply:
This is handleCall<RequestType, ResponseType>

`  type handleCall<RequestType, ResponseType> =
    handleUnaryCall<RequestType, ResponseType> |
    handleClientStreamingCall<RequestType, ResponseType> |
    handleServerStreamingCall<RequestType, ResponseType> |
    handleBidiStreamingCall<RequestType, ResponseType>;`

But I doubt is needed for this doc cause is hard. If you think that it needs to be added then I will

Reply:
fixed